config/routes.rb

```diff
@@ -0,0 +1,9 @@
+Rails.application.routes.draw do
+  # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
+
+  # Defines the root path route ("/")
+  # root "articles#index"
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show]
+  end
```

Comment:
✅ Has users and posts routes added to config/routes.rb and maintained a RESTful URL 👍🏻